app/controllers/api/messages_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::MessagesController < ApplicationController
+  def index
+    @group = Group.find(params[:group_id])
```

Comment:
@group使ってますか？？

Reply:
```suggestion
    @group = Group.find(params[:group_id])
```
groupのIDの特定に使うような気持ちで書いたようです、、
消してみても問題なく動いたので削除いたしました。